recipes/stella-cv-fbow/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 3.15)
+project(test_package LANGUAGES C)
+project(test_package LANGUAGES CXX)
```

```suggestion
project(test_package LANGUAGES CXX)
```

There seems to be no c file.